### PR TITLE
Add more logging

### PR DESCRIPTION
**Pull request by camjackson**
Also includes some misc. tech tasks e.g. upgrading dependencies.

### CODE REVIEW

**Comment by camjackson** [comment]
@ineffyble hopefully this logging will make it easier to debug. Just looking at the code, I think I see a problem with the way we map over the records asynchronously, but I *don't* think that's what's causing the bug we were looking at yesterday.

File: bin/lambda-deploy.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 set -e
 
 echo 'REMOVING DEV DEPENDENCIES'
-npm prune --production
+yarn --production
```
Comment (ineffyble) [inline bin/lambda-deploy.sh:6]: Should this be `yarn install`? or does `yarn` default to install?

Comment (camjackson) [reply bin/lambda-deploy.sh:6]: Yep, default command is install.

**Review by ineffyble — APPROVED** [review]
LGTM

**Pull request merged**